app/assets/javascripts/client/directives/learning_questions/learningQuestionModal.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .directive('learningQuestionModal', learningQuestionModal);
+
+  function learningQuestionModal() {
+    return {
+      restrict: 'E',
+      scope: {
+        reminder: '@'
+      },
+      templateUrl: 'client/views/modals/learning_questions/add_learning_question.html',
+      link: function(scope) {
+        // The modal exists on the parent scope, so we have to reach up to it  to bind any closing actions on the modal.
+        var parentScope = scope.$parent;
```

Comment:
I've went back to this code and looked a bit closer at what it would take to convert this over to using events.

There are about four different spots that this modal is being created in which emitting the event would require us to place the same code in four different spots:

```
$scope.$on('close-learning-question-modal', function() {
  $scope.modal.dismiss('cancel');
});
```

The event code isn't complex but the number of places that this has to be placed in is higher than I'd like it to be.